iphone/Classes/TiUIiOSStepperProxy.m

```diff
@@ -35,15 +31,10 @@ -(TiDimension)defaultAutoHeightBehavior:(id)unused
 }
 #endif
 
--(UIView *)parentViewForChild:(TiViewProxy *)child
-{
-    return [[(TiUIiOSStepper*)[self view] stepper] superview];
-}
 
 -(NSNumber*)value
 {
-    NSNumber *value = [NSNumber numberWithDouble:[[(TiUIiOSStepper*)[self view] stepper] value]];
-    return value;
+    return NUMDOUBLE([(TiUIiOSStepper*)[self view] getValue]);
```

Comment:
Ok, here we use it. You could just return `NUMDOUBLE([[self stepper] value])`, or am I missing something?